frontend/src/utils/store.ts

```diff
@@ -2,13 +2,21 @@ import { useMainStore } from '@/stores'
 import { db } from './db'
 import { v4 as uuidv4 } from 'uuid'
 
+export const examplesNames = [
+  'football_prediction_market.py',
+  'llm_erc20.py',
+  'storage.py',
+  'user_storage.py',
+  'wizard_of_coin.py'
+]
+
 export const setupStores = async () => {
   const mainStore = useMainStore()
-  // await db.contractFiles.clear()
-  // localStorage.setItem('mainStore.contractsModified', '')
+  const contracts = await db.contractFiles.toArray()
   if (
-    (await db.contractFiles.count()) === 0 &&
-    !localStorage.getItem('mainStore.contractsModified')
+    contracts
+      .filter((c) => (c.example && !c.updatedAt) || examplesNames.includes(c.name))
+      .map((c) => c.id).length === 0
```

Comment:
@bradleySuira what is this map doing? do we need it?

Reply:
Also, the comparison with the name doesn't seem reliable for this decision

Reply:
@cristiam86 
- removed map we don't need it
- we need to keep the names just in case for old browser storage because now we are adding the example field inside the db
-